### PR TITLE
Add svg support and local anchors

### PR DESCRIPTION
**Pull request by kpurrmann**

### CODE REVIEW

File: Classes/View/PdfView.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ private function replaceStrings($content) {
             }
         }
 
-        return $content;
+        return preg_replace('/href=\"(.*?)#(.*?)\"/', 'href="#$2"', $content);
```
Comment (martin-helmich) [inline Classes/View/PdfView.php]: Won't this break links to actual external URLs? For example when `a.html` is printed and contains a link to `b.html#foo`?

Comment (kpurrmann) [inline Classes/View/PdfView.php]: Yes you're right. May I should replace against current uri.

Comment (martin-helmich) [inline Classes/View/PdfView.php]: Sounds good to me! :+1:

**Comment by martin-helmich** [comment]
I'm still having a few difficulties understanding how links in the HTML/PDF document should be handled in general.

Here's just some thoughts on how I intuitively think this should work:

1. **Links to external documents on the same domain.** In general, I suppose it would be best if all `href`'s in the HTML document were absolute so that they still work when the document is rendered to an offline PDF (i.e. convert `<a href="/index.php?id=123">` to `<a href="http://example.com/index.php?id=123>`). I'd bet money that there's some obscure TypoScript option that does exactly that.
2. **Intra-document links.** These should either contain only the fragment part of the URL (which should work in the PDF without any hassle) or the exact same document URL + fragment part (in which case the document URL should be stripped out).

@pstranghoener, it that assumption correct? What's the intended behaviour here?

**Comment by kpurrmann** [comment]
See #29

**Pull request closed without merging**